Sources/Runner/Commands/Edit.swift

```diff
@@ -5,14 +5,21 @@ import Files
 import MarathonCore
 
 func editDanger() throws -> Void {
-    
-    // Exit if a dangerfile was not found at any supported path
-    guard let dangerfilePath = Runtime.getDangerfile() else {
-        print("Could not find a Dangerfile")
-        print("Please use a supported path: \(Runtime.supportedPaths)")
-        exit(1)
+
+    let createDangerfile = { () -> String in
+        do {
+            let template = "import Danger \n let danger = Danger()"
+            let data = template.data(using: .utf8)!
+            return try FileSystem().createFile(at: "Dangerfile.swift", contents: data).path
+        } catch {
+            print("Could not find or generate a Dangerfile")
```

Comment:
Maybe print the error while you're at it?

Reply:
I thought about this but wasn't sure if it was right to propagate error messages coming from the `Files` module. I didn't want to clog up stdout